privaterelay/management/commands/update_phone_remaining_stats.py

```diff
@@ -23,6 +23,13 @@ def update_phone_remaining_stats():
     updated_profiles = []
     for social_account in accounts_with_phones:
         profile = social_account.user.profile
+        if not profile.date_phone_subscription_checked:
+            first_billing_date = profile.date_subscribed_phone + timedelta(
```

Comment:
`profile.date_subscribed_phone` is nullable, and may be unset for some users since we added it in the middle of the phones feature. I think we skip this path if it is null, something like:

```python
if (not profile.date_phone_subscription_checked) and (profile.date_subscribed_phone is not null):
```